engine/src/Format.inl

```diff
@@ -1,5 +1,8 @@
 #include "Engine/Core/Tools/Format.hpp"
 
+#include <cassert>
+#include <memory>
```

Comment:
I would rather see all the headers together next to each others, rather than split